lib/plugins/aws/package/compile/events/cloud-front.js

```diff
@@ -128,6 +128,7 @@ class AwsCompileCloudFrontEvents {
         FieldLevelEncryptionId: { type: 'string' },
         OriginRequestPolicyId: { type: 'string' },
         SmoothStreaming: { type: 'boolean' },
+        ResponseHeadersPolicyId: { type: 'string' },
```

Comment:
Hey @benheymink 👋 I think it would be awesome to add a test that confirms this property is recognized without issues

Reply:
Update - nevermind, found them. (`test/unit/lib/plugins/aws/package/compile/events/cloud-front.test.js` will add a sanity check.

Reply:
@pgrzesik Yeah still no luck on those tests, and I don't really want to perform major heart surgery to your test structure....